app/src/main/java/org/schabi/newpipe/player/ServicePlayerActivity.java

```diff
@@ -690,8 +691,10 @@ private void onPlayModeChanged(final int repeatMode, final boolean shuffled) {
 
     private void onPlaybackParameterChanged(final PlaybackParameters parameters) {
         if (parameters != null) {
-            playbackSpeedButton.setText(formatSpeed(parameters.speed));
-            playbackPitchButton.setText(formatPitch(parameters.pitch));
+            if (menu != null && player != null) {
+                MenuItem item = menu.findItem(R.id.action_playback_speed);
```

Comment:
Please make this variable `final`

Reply:
done

Reply:
Instead of saying "done", please click on "Resolve conversation" next time ;)